File: src/modify.js
const whileToFor = () => {
  let i = 0;
  while (i < 5) {
    console.log(i);
    i++;
  }
};

const continueGuardClause = () => {
  for (let i = 1; i < 5; i++) {
    // Add a guard clause to skip 2 and 3
    console.log("Sure glad this isn't 2 or 3");
    console.log(i);
  }
};

module.exports = {
  whileToFor,
  continueGuardClause,
};

File: src/modify.py
def while_to_for():
    i = 0
    while i < 5:
        print(i)
        i += 1


def continue_guard_clause():
    for i in range(1, 5):
        # Add a guard clause to skip 2 and 3
        print("Sure glad this isn't 2 or 3")
        print(i)
